src/app/trainees/reset-trainee-list/reset-trainee-list.component.ts

```diff
@@ -24,9 +24,9 @@ export class ResetTraineeListComponent {
   constructor(private store: Store, private traineeService: TraineeService) {}
```

Comment:
The clear / reset functionality seems not to set to default which I thought was sortColumn=submissionDate&sortOrder=desc&pageNumber=0